__tests__/commands/global.js

```diff
@@ -22,31 +22,31 @@ async function runGlobal(
   name: string,
   checkGlobal?: ?(config: Config, reporter: Reporter, out: string) => ?Promise<void>,
 ): Promise<void> {
+  let out = '';
+  const stdout = new stream.Writable({
```

Comment:
Yeah i noticed this too, I'm going to abstract it out in a subsequent pull request.